ansible/roles/common/tasks/main.yml

```
@@ -0,0 +1,136 @@
+  tags: common
+
+- name: Add project user to sudo group
+  user:
+    name: "{{ project_user }}"
+    groups: sudo
+    append: yes
+  when: project_user != "root"
+
+- name: Update system packages
+  apt:
+    update_cache: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)